packages/babel-helper-transform-fixture-test-runner/src/index.ts

```diff
@@ -441,6 +441,13 @@ function normalizeOutput(code: string, normalizePathSeparator?: boolean) {
       );
     }
   }
+
+  // In Babel 8, preset-env logs transform- instead of proposal-. Manually rewrite
+  // the output logs so that we don't have to duplicate all the debug fixtures for
+  // the two different Babel versions.
+  // TODO(Babel 8): Remove this.
+  result = result.replace(/(\s+)proposal-/gm, "$1transform-");
```

Comment:
The `normalizeOutput` is also applied to the transformed code. Can we make a new flag, e.g. `normalizePresetEnvDebug` like we did for `normalizePathSeparator`? Then we can enable this flag only when we are validating logs, preferably from `preset-env` only. So the fixture runner won't accidentally replace `proposal-` to `transform-` out of the preset-env scope.